app/assets/v2/js/grants/cancel_subscription.js

```diff
@@ -67,16 +67,17 @@ window.onload = function() {
                 ).send({from: accounts[0], gasPrice: 4000000000})
                   .on('confirmation', function(confirmationNumber, receipt) {
                     console.log('receipt', receipt);
-
                     form.submit();
                   });
               });
             })
             .on('confirmation', function(confirmationNumber, receipt) {
               console.log('receipt', receipt);
+              window.location = $('#grant-link').val();
             })
             .on('error', function(err) {
               console.log('err', err);
+              // TODO: Redirect to same page and throw error ?
```

Comment:
maybe at least throw an error and log something to sentry?

Reply:
like an alert?